.npmignore

```diff
@@ -7,6 +7,6 @@
 /tmp/
 /.*
 *.log
-!/.travis.yml
+!/.github/workflows/ci.yml
```

Comment:
Maybe a comment for future travellers why this is in here?

Reply:
Great idea! Updated!